sdk/keyvault/keyvault-certificates/test/internal/challengeBasedAuthenticationPolicy.spec.ts

```diff
@@ -9,7 +9,7 @@ import {
   AuthenticationChallengeCache,
   AuthenticationChallenge,
   parseWWWAuthenticate
-} from "../../../keyvault-common/src";
+} from "@azure/core-http";
```

Comment:
if we're testing the policy itself (as the test name indicates), this test file should live in core-http as well.

Reply:
Yup, this is repeated in the three clients, so we could clean that up too (just by removing the files with the same name, challengeBasedAuthenticationPolicy.spec.ts).